public/app/features/integrated-alerting/components/Table/Table.utils.ts

```diff
@@ -0,0 +1,4 @@
+import { PAGE_SIZES } from './Table.constants';
+
+export const getProperPageSize = (pageSize: number) =>
+  PAGE_SIZES.find(p => p.value === pageSize) ? pageSize : PAGE_SIZES[0].value;
```

Comment:
```suggestion
  PAGE_SIZES.find(size => size.value === pageSize) ? pageSize : PAGE_SIZES[0].value;
```

nit: please, try to avoid single letter variable they're not indexes.

Reply:
👍